content/en/synthetics/ci.md

```diff
@@ -526,18 +584,46 @@ The execution rule associated with the test is always the most restrictive one t
 
 You can configure on which url your test starts by providing a `startUrl` to your test object and build your own starting URL using any part of your test's original starting URL and the following environment variables:
 
-| Environment variable | Description                  | Example                                                |
-|----------------------|------------------------------|--------------------------------------------------------|
-| `URL`                | Test's original starting URL | `https://www.example.org:81/path/to/something?abc=123` |
-| `DOMAIN`             | Test's domain name           | `example.org`                                          |
-| `HOST`               | Test's host                  | `www.example.org:81`                                   |
-| `HOSTNAME`           | Test's hostname              | `www.example.org`                                      |
-| `ORIGIN`             | Test's origin                | `https://www.example.org:81`                           |
-| `PARAMS`             | Test's query parameters      | `?abc=123`                                             |
-| `PATHNAME`           | Test's URl path              | `/path/to/something`                                   |
-| `PORT`               | Test's host port             | `81`                                                   |
-| `PROTOCOL`           | Test's protocol              | `https:`                                               |
-| `SUBDOMAIN`          | Test's sub domain            | `www`                                                  |
+
+`URL`
+: Test's original starting URL <br>
+**Example**: `https://www.example.org:81/path/to/something?abc=123`
+
+`DOMAIN`
+: Test's domain name<br>
+**Example**: `example.org`
+
+`HOST`
+: Test's host<br>
+**Example**: `www.example.org:81`
+
+`HOSTNAME`
+: Test's hostname<br>
+**Example**: `www.example.org`
+
+`ORIGIN`
+: Test's origin<br>
+**Example**: `https://www.example.org:81`
+
+`PARAMS`
+: Test's query parameters<br>
+**Example**: `?abc=123`
+
+`PATHNAME`
+: Test's URl path<br>
+**Example**: `/path/to/something`
+
+`PORT`
+: Test's host port<br>
+**Example**: `81`
+
+`PROTOCOL`
+: Test's protocol<br>
+**Example**: `https:`
+
+`SUBDOMAIN`
+: Test's sub domain<br>
+**Example**: `www`
```

Comment:
It looks like the execution image + the three images after this table are broken in the preview, but loading in the live docs: https://docs.datadoghq.com/synthetics/ci/?tab=yarn